src/introspect/introspect.ts

```diff
@@ -67,6 +67,26 @@ export class Introspect {
     return query.makeExplain();
   }
 
+  static getLimit0QueryColumnIntrospectionQuery(
+    query: SqlQuery,
+    context: Record<string, any>,
+    timeZone?: string,
+    sqlTypesHeader?: boolean,
```

Comment:
Would we ever call this function without setting this to `true`? Is there a valid use case for leaving this undefined/false?